### PR TITLE
feat(DevTools): Can be edit Brush if is Property is Null

### LINKED ISSUE #10659: [DevTools] Can be edit Brush if is Property is Null
**To Reproduce**

1. Create Project form template
2. Replace the content of `MainWindow.axaml' like this:
   ```xaml
    <Border Background="{x:Null} Width="20" Height="20" BorderThickness="1" BorderBrush="Black"/>    
   ```
4. Run
5. Lunch DevTools
6. Select Border
7. Try to edit Background

**Expected behavior**
Click on value show ColorPicker

**Screenshots**

**Desktop (please complete the following information):**

- OS: All
- Version: cbe0bb861a767b7bf7b268bbaa1d89dcf8523aea

**Additional context**
none

cc: @pr8x

### PR DESCRIPTION
**Pull request by workgroupengineering**
## What does the pull request do?

DevTools

Allow DevTools to:
- Edit null brush property 
- Reset to null brush Property 
- Show color name

## What is the current behavior?

Can not be edit Brush if is Property is Null

## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation

## Fixed issues

Fixes #10659

### CODE REVIEW

File: src/Avalonia.Diagnostics/Diagnostics/Controls/BrushEditor.axaml.cs [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+            context.DrawText(ft,
+                new Point(Bounds.Width / 2 - ft.Width / 2, Bounds.Height / 2 - ft.Height / 2));
+        }
+
+        /// <summary>
+        /// <see cref="https://stackoverflow.com/questions/6763032/how-to-pick-a-background-color-depending-on-font-color-to-have-proper-contrast">How to pick a background color depending on font color to have proper contrast</see>/>
+        /// </summary>
+        /// <param name="brush"></param>
+        /// <returns></returns>
+        private static IBrush GetTextBrush(IBrush brush)
```
Comment (robloo) [inline src/Avalonia.Diagnostics/Diagnostics/Controls/BrushEditor.axaml.cs:110]: This should probably use the same algorithm we have for relative luminance. It's what is used in WinUI itself was copied over and used in the ColorPicker itself.

https://github.com/AvaloniaUI/Avalonia/blob/db67ad1f8776b5d1b6d13b139c6e3f464d70e54f/src/Avalonia.Controls.ColorPicker/Helpers/ColorHelper.cs#L26

Used in a very similar way to what you are already doing here.

https://github.com/AvaloniaUI/Avalonia/blob/db67ad1f8776b5d1b6d13b139c6e3f464d70e54f/src/Avalonia.Controls.ColorPicker/Converters/ContrastBrushConverter.cs#L64

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0036325-beta`. (feed url: https://pkgs.dev.azure.com/AvaloniaUI/AvaloniaUI/_packaging/avalonia-all/nuget/v3/index.json) [PRBUILDID]

File: src/Avalonia.Diagnostics/Diagnostics/Controls/BrushEditor.axaml.cs [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+using Avalonia.Media;
+using Avalonia.Media.Immutable;
+
+namespace Avalonia.Diagnostics.Controls
+{
+    [TemplatePart("PART_ClearButton", typeof(Button))]
+    partial class BrushEditor : TemplatedControl
+    {
+        private readonly EventHandler<RoutedEventArgs> clearHandler;
+        private Button? _cleraButton = default;
```
Comment (robloo) [inline src/Avalonia.Diagnostics/Diagnostics/Controls/BrushEditor.axaml.cs:17]: Spelling : clearButton

**Review by maxkatz6 — APPROVED** [review]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0036356-beta`. (feed url: https://nuget-feed-all.avaloniaui.net/v3/index.json) [PRBUILDID]

**Pull request merged**